main/src/cgeo/geocaching/location/Deg2UTM.java

```diff
@@ -0,0 +1,80 @@
+package cgeo.geocaching.location;
+
+/**
+ * Source: https://stackoverflow.com/questions/176137/java-convert-lat-lon-to-utm
+ */
+public class Deg2UTM {
+    private double easting;
+    private double northing;
+    private final int zone;
+    private final char letter;
+
+    public Deg2UTM(final double lat, final double lon) {
+        zone = (int) Math.floor(lon / 6 + 31);
+        if (lat < -72) {
```

Comment:
It should be possible to replace this with an array/string index (divide by 8 ...)